dek-registry/generated/swagger-ui/dek-registry-api-spec.yaml

```diff
@@ -6,7 +6,7 @@ paths:
   /dek-registry/v1/keks/{name}/deks:
     get:
       summary: Get a list of dek subjects.
-      operationId: getDekSubjects
+      operationId: getDekSubjects_1
```

Comment:
See if you can regenerate so that it doesn't add the `_1` suffix.  Otherwise you can fix them by hand